tests/Doctrine/Tests/ODM/PHPCR/Functional/Translation/AttributeTranslationStrategyTest.php

```diff
@@ -118,6 +118,74 @@ public function testLoadTranslation()
         $this->assertEquals(array(), $doc->getSettings());
     }
 
+    public function testSubRegionSaveTranslation()
+    {
+        // First save some translations
+        $data = array();
+        $data['topic'] = 'Some interesting subject';
+        $data['text'] ='Lorem ipsum...';
+
+        $node = $this->getTestNode();
+
+        $strategy = new AttributeTranslationStrategy($this->dm);
+        $strategy->saveTranslation($data, $node, $this->metadata, 'en');
+
+        // Save translation in another language
+
+        $data['topic'] = 'Some interesting american subject';
+
+        $strategy->saveTranslation($data, $node, $this->metadata, 'en_US');
+        $this->dm->flush();
+
+        // Then test we have what we expect in the content repository
+        $node = $this->session->getNode('/' . $this->testNodeName);
+
+        $this->assertTrue($node->hasProperty(self::propertyNameForLocale('en', 'topic')));
+        $this->assertTrue($node->hasProperty(self::propertyNameForLocale('en_US', 'topic')));
```

Comment:
do you propose to go for uppercase countries? i don't know which is better, but we should be strict about it. i think dan's expression only allows lower case. either way i would add a test case for that.

btw, is it granted that there can only be two letter language and country names, or are there longer codes we need to respect? i.e. `zh-Hant` for chinese traditional...